pom.xml

```diff
@@ -3,11 +3,38 @@
    http://maven.apache.org/xsd/maven-4.0.0.xsd">
   <modelVersion>4.0.0</modelVersion>
 
+  <name>Heterogeneous-Memory Storage Engine Java Bindings</name>
+  <description>Fast embeddable key-value store designed for SSDs and persistent memory</description>
+  <url>https://hse-project.github.io/</url>
```

Comment:
Help is mainly needed here. Is this what we want?

Reply:
Heh, I see where you got that "fast embeddable" from, but I think it probably should have a comma after the "fast" :-)

Reply:
I just ripped it off the website.

> HSE is a fast embeddable key-value store designed for SSDs and persistent memory. HSE optimizes performance and endurance by orchestrating data placement across DRAM and multiple classes of solid-state storage devices.

- https://github.com/hse-project/hse-python/blob/master/pyproject.toml#L6
- https://github.com/hse-project/hse/blob/master/lib/meson.build#L179

Should probably try to consolidate all this wording.

Reply:
Search "fast embeddable" and you'll find many projects with these adjectives, and it's a 50/50 mix of whether or not there is a comma.  Since we don't use a comma anywhere else I wouldn't use it here.  The alternative is to change it everywhere else which doesn't seem worth the effort.